.env.bak

```diff
@@ -0,0 +1,12 @@
+REDDIT_USERNAME=PYT_Verification_Bot
```

Comment:
`.env.bak` needs to be added to the `.gitignore` file. We don't want the keys to be available to the public